pkgs/tools/inputmethods/libinput-gestures/default.nix

```diff
@@ -0,0 +1,35 @@
+{lib, stdenv, fetchFromGitHub, bash, fakeroot, libinput, xdotool, makeWrapper}:
+stdenv.mkDerivation rec {
+  pname = "libinput-gestures";
+  version = "2.32";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "bulletmark";
+    repo = "libinput-gestures";
+    rev = version;
+    sha256 = "1by6sabx0s8sd9w5675gc26q7yccxnxxsjg4dqlb6nbs0vcg81s7";
+  };
+
+  buildInputs = [ bash fakeroot makeWrapper ];
+
+  installPhase =
+    ''
+      mkdir -p $out
+      fakeroot -- bash libinput-gestures-setup -d "$out" install
```

Comment:
Do you really need ```fakeroot```?

Reply:
Yes, libinput-gestures-setup bails if it's not executed as root, see https://github.com/bulletmark/libinput-gestures/blob/f3c0e61f5b85073eee3ddfe0216f8dd040ddf64d/libinput-gestures-setup#L145-L148

Reply:
I would probably just patch that out.

Reply:
Turns out I was a moron and it's not required. Removing it now.